### PR TITLE
Add single document feature

### PR DESCRIPTION
**Pull request by JonatascNascimento**

### CODE REVIEW

File: lib/validates_identity/usage.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      @identity_type = identity_type
+      @only = only
+    end
+
+    def valid?
+      return false if identity_type.blank? && only.blank?
+      return false if identity_type.blank? && %i[person legal].include?(only)
+
+      identity_type.blank? || [:person, :legal, nil].include?(only)
+    end
```
Comment (plribeiro3000) [inline lib/validates_identity/usage.rb]: What if we improve this code a bit?

Something like

```ruby
if identity_type.present? && [:person, :legal, nil].include?(only)
  return true
elsif identity_type.blank? && ValidatesIdentity.identity_type_aliases.keys.include?(only)
  return true
else
  false
end
```

WDYT?

File: lib/validates_identity/usage.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class ValidatesIdentity
+  class Usage
```
Comment (plribeiro3000) [inline lib/validates_identity/usage.rb]: `DslUsage`? WDYT?

File: lib/validates_identity/usage.rb (hunk 1 above)
Comment (JonatascNascimento) [reply lib/validates_identity/usage.rb]: I agree with it, it makes it more clear.

File: lib/validates_identity/usage.rb (hunk 2 above)
Comment (JonatascNascimento) [reply lib/validates_identity/usage.rb]: What about `Definition`?

Comment (plribeiro3000) [reply lib/validates_identity/usage.rb]: Sure! Its way better! Makes perfect sense to me! 💥

Comment (JonatascNascimento) [reply lib/validates_identity/usage.rb]: Great! I'll commit it soon.

File: lib/validates_identity/usage.rb (hunk 1 above)
Comment (JonatascNascimento) [reply lib/validates_identity/usage.rb]: Commited

**Review by plribeiro3000 — APPROVED** [review]
🚀 🚀 🚀 🚀 🚀

**Pull request merged**